ai21_tokenizer/jurassic_tokenizer.py

```diff
@@ -19,11 +19,16 @@ class SpaceSymbol:
 class JurassicTokenizer(BaseTokenizer):
     def __init__(
         self,
-        model_path: PathLike,
+        model_path: Optional[PathLike] = None,
+        model_file_handle: Optional[BinaryIO] = None,
         config: Optional[Dict[str, Any]] = None,
     ):
+        JurassicTokenizer._assert_exactly_one(model_path, model_file_handle)
+
+        model_proto = load_binary(model_path) if model_path else model_file_handle.read()
```

Comment:
model_proto? you mean `model_prototype` ? if so please rename :)

Reply:
I meant `model_proto` because that's the arg name to `spm.SentencePieceProcessor`